README.md

```diff
@@ -52,7 +52,7 @@ For command line help, visit our wiki page on Bourbon’s [command line interfac
 
   It’s not recommended to add or modify the Bourbon files so that you can update them easily.
```

Comment:
No, because we still allow people to install the Bourbon gem to their machine, and then just use our CLI to install a local instance of Bourbon into a project. This is for people who may not be in a Rails project, or a project with npm…but maybe just a simple website with no package manager at all.